conf/messages

```diff
@@ -659,6 +659,9 @@ fh.business.partners.enterAddress.line3.error.pattern = Address lines must not b
 fh.business.partners.enterAddress.postcode.label = Postcode (optional)
 fh.business.partners.enterAddress.postcode.error.pattern = Enter a valid postcode
 
+fh.businessPartners.partnership.confirmRegisteredAddress.title=Confirm the partnership’s address
```

Comment:
The title should be "Confirm the partnership’s registered office address?" according to the ticket